### ISSUE TITLE
Sending cached envelopes created by sentry_handle_exception

### ISSUE THREAD

**Issue by hasore** [getsentry/sentry-native#815]
How to drain the cached envelopes created by sentry_handle_exception?
Needs to wait for application restart or there's a better behaviour to send the cached envelopes created?

**Comment by Swatinem** [comment]
Yes, a restart of the application (or just the SDK via `sentry_init`) would be needed, as `sentry_init` will iterate over all the cached envelopes on disk and submit them for sending.

I’m wondering how you ended up in this situation. Envelopes should only be cached on disk from within the signal handler, which operates under the assumption that the application is indeed terminating after handling a hard crash.
